### PR TITLE
New Holidays: The Great War, D&D, and Species Celebrations

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Adds some new holidays.

- Remembrance Day is self explanatory, and falls on November 11th.
- UN Day is on the anniversary of the foundation of the UN, October 24th.
- Wizards Day is on Gary Gygax's birthday, July 27th.
- Tiziran Unification Day falls on September 1st, and is the date on which Lizards were made a roundstart race.
- The Festival of Atrakor's Might falls on June 15th, and is the date on which the Lizard visual overhaul was merged, bringing horns, frills, and spines to make lizards less bland.
- Draconic Day falls on May 3rd, and is the date on which the Draconic language was merged, heralding the rise of the Lizard metagangs (or not).
- Fleet Day is the Moths' special day, falling on Jan 19th which is the date on which Moths were merged.
- Finally, the Festival of Holy Lights is the Ethereal holiday, falling on Nov 28th, the date on which Ethereals were merged.

_Taking bets now on whether oranges closes this PR:_
orangemanunhappy

## Why It's Good For The Game
The human holidays are ones which I was quite surprised to find not already mentioned, given their significance to both European countries and world history. Wizard day is just a bit of fun. And finally, I think the species holidays bring some much needed flavour to the world.

## Changelog
:cl: Inept
add: Some new holidays have been added.
/:cl:

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -661,3 +661,102 @@
...
+/datum/holiday/victory_day/getStationPrefix()
+	return pick("Liberty", "Victory", "Peace")
+
+/datum/holiday/un_day
+	name = "Anniversary of the Foundation of the United Nations"
+	begin_month = OCTOBER
+	begin_day = 24
+
+/datum/holiday/un_day/greet()
+	return "On this day in 1945, the United Nations was founded, laying the foundation for humanity's united government!"
```
Comment (maxymax13) [inline code/modules/holiday/holidays.dm:688]: corn syrup

**Comment by YakumoChen** [comment]
Catgirl day catgirl day

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -661,3 +661,102 @@
...
+	begin_day = 11
+	drone_hat = /obj/item/food/grown/poppy
+
+/datum/holiday/remembrance_day/getStationPrefix()
+	return pick("Peace", "Armistice", "Poppy")
+
+/datum/holiday/victory_day
+	name = "Victory in Europe Day"
+	begin_month = MAY
+	begin_day = 8
```
Comment (interestingusernam3) [inline code/modules/holiday/holidays.dm:677]: Violates https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#banned-content - this is National Socialist Party of Germany related content.

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/modules/holiday/holidays.dm:688]: But there is no united human goverment in the lore?

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (YakumoChen) [reply code/modules/holiday/holidays.dm:677]: Wasn't even in the PR body, trying to hide this one lol

File: code/modules/holiday/holidays.dm [hunk 3]
```diff
@@ -661,3 +661,102 @@
...
+/datum/holiday/remembrance_day
+	name = "Remembrance Day"
+	begin_month = NOVEMBER
+	begin_day = 11
+	drone_hat = /obj/item/food/grown/poppy
+
+/datum/holiday/remembrance_day/getStationPrefix()
+	return pick("Peace", "Armistice", "Poppy")
+
+/datum/holiday/victory_day
```
Comment (optimumtact) [inline code/modules/holiday/holidays.dm:674]: remove this holiday

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (EOBGames) [reply code/modules/holiday/holidays.dm:677]: I uh, would have thought the day celebrating victory over the Nazis would have been kosher, but if it's not, I'll remove it

Comment (EOBGames) [reply code/modules/holiday/holidays.dm]: > Wasn't even in the PR body, trying to hide this one lol

it's also mentioned in the PR body:
"Remembrance Day and **VE Day** are self explanatory, and fall on November 11th and May 8th, respectively."
VE Day, May 8th, is the celebration of the end of WW2, and was, I thought, self explanatory enough to not require its own callout, same with Remembrance Day.

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (EOBGames) [reply code/modules/holiday/holidays.dm:680]: > But there is no united human goverment in the lore?

There is, see https://github.com/tgstation/common_core/blob/master/Governments.md

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (YakumoChen) [reply code/modules/holiday/holidays.dm]: You are assuming a proud American keeps track of bank days not in the greatest nation on Earth

File: code/modules/holiday/holidays.dm [hunk 4]
```diff
@@ -661,3 +661,94 @@
...
+	return pick("Draconic", "Literature", "Reading")
+
+//Fleet Day is celebrated on Jan 19th, the date on which moths were merged (#34498)
+/datum/holiday/fleet_day
+	name = "Fleet Day"
+	begin_month = JANUARY
+	begin_day = 19
+
+/datum/holiday/fleet_day/greet()
+	return "This day commemorates another year of successful survival aboard the Mothic Grand Nomad Fleet. Moths galaxywide are encouraged to eat, drink, and be merry."
```
Comment (optimumtact) [inline code/modules/holiday/holidays.dm:739]: drink what?

Comment (EOBGames) [reply code/modules/holiday/holidays.dm:739]: never heard the phrase "to eat, drink, and be merry" before?

Comment (optimumtact) [reply code/modules/holiday/holidays.dm:739]: what do they drink EOB, what do they drink

Comment (YakumoChen) [reply code/modules/holiday/holidays.dm:739]: moth milk

Comment (EOBGames) [reply code/modules/holiday/holidays.dm:739]: I guess I'll need to wrap up the lizard food PR and move onto moths to answer that question, huh?

**Pull request merged**